include/cantera/kinetics/RxnRates.h

```diff
@@ -91,6 +91,10 @@ class Arrhenius
         return m_E;
     }
 
+    //! Check to make sure that the sticking coefficient is
+    //! less than 1 over a range of temperatures
+    void validate(const std::string& equation);
```

Comment:
I do not believe that this is the correct place to add this check, as for the vast majority of `Arrhenius` rates the sticking coefficient does not apply.